.github/workflows/actionlint.yml

```
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: Apache-2.0
+
+name: Lint GitHub Actions workflows
+on:
+  push:
+    branches:
+      - "main"
+    paths:
+      - '.github/**'
```

```suggestion
      - '.github/workflows/*.ya?ml'
```

seems more correct since this is a workflow yaml linter, no?

---

Works for me @bjhargrave!